source/project.json

```diff
@@ -1,4 +1,24 @@
 {
   "current_version": "1.4.3",
-  "current_v8": "4.1.0.21"
+  "current_v8": "4.1.0.21",
+  "languages": [
+    {"code": "cn", "name": "中文", "name-en": "Chinese"},
+    {"code": "de", "name": "Deutsch", "name-en": "German"},
+    {"code": "el", "name": "Ελληνικά", "name-en": "Greek"},
+    {"code": "en", "name": "English", "name-en": "English"},
+    {"code": "es", "name": "Español", "name-en": "Spanish"},
+    {"code": "fi", "name": "Suomi", "name-en": "Finnish"},
+    {"code": "fr", "name": "Français", "name-en": "French"},
+    {"code": "he", "name": "עברית", "name-en": "Hebrew", "rtl": true},
+    {"code": "id", "name": "Bahasa Indonesia", "name-en": "Indonesian"},
+    {"code": "it", "name": "Italiano", "name-en": "Italian"},
+    {"code": "ja", "name": "日本語", "name-en": "Japanese"},
+    {"code": "ko", "name": "한국어", "name-en": "Korean"},
+    {"code": "pt_BR", "name": "Português (BR)", "name-en": "Portuguese (Brazil)"},
+    {"code": "pt_PT", "name": "Português (PT)", "name-en": "Portuguese (Portugal)"},
+    {"code": "ru", "name": "Русский", "name-en": "Russian"},
+    {"code": "tr", "name": "Türkçe", "name-en": "Turkish"},
+    {"code": "uk", "name": "Українська", "name-en": "Ukrainian"},
+    {"code": "zh_TW", "name": "台灣", "name-en": "Chinese (Taiwan)"}
```

Comment:
Ukrainian is ok.

On Wednesday, March 4, 2015, Hsiaoming Yang <notifications@github.com>
wrote:

> In source/project.json
> <https://github.com/iojs/website/pull/258#discussion_r25753143>:
>
> > +    {"code": "el", "name": "Ελληνικά", "name-en": "Greek"},
> > +    {"code": "en", "name": "English", "name-en": "English"},
> > +    {"code": "es", "name": "Español", "name-en": "Spanish"},
> > +    {"code": "fi", "name": "Suomi", "name-en": "Finnish"},
> > +    {"code": "fr", "name": "Français", "name-en": "French"},
> > +    {"code": "he", "name": "עברית", "name-en": "Hebrew", "rtl": true},
> > +    {"code": "id", "name": "Bahasa Indonesia", "name-en": "Indonesian"},
> > +    {"code": "it", "name": "Italiano", "name-en": "Italian"},
> > +    {"code": "ja", "name": "日本語", "name-en": "Japanese"},
> > +    {"code": "ko", "name": "한국어", "name-en": "Korean"},
> > +    {"code": "pt_BR", "name": "Português (BR)", "name-en": "Portuguese (Brazil)"},
> > +    {"code": "pt_PT", "name": "Português (PT)", "name-en": "Portuguese (Portugal)"},
> > +    {"code": "ru", "name": "Русский", "name-en": "Russian"},
> > +    {"code": "tr", "name": "Türkçe", "name-en": "Turkish"},
> > +    {"code": "uk", "name": "Українська", "name-en": "Ukrainian"},
> > +    {"code": "zh_TW", "name": "台灣", "name-en": "Chinese (Taiwan)"}
>
> @popomore <https://github.com/popomore> @PeterDaveHello
> <https://github.com/PeterDaveHello> 正體的說法不太友好，「簡體」對應的應該是「繁體」
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/iojs/website/pull/258/files#r25753143>.
>